internal/models/cli.go

```diff
@@ -70,7 +70,7 @@ type RunOptions struct {
 	TargetNode        string
 	WorkloadUrl       *url.URL
 	Name              string
-	WorkloadType      string
+	WorkloadType      NexExecutionProvider
```

Comment:
I would vote to rename `NexExecutionProvider` here to `NexWorkloadType`.

Reply:
if that'll make you happy...ill do it